cocojson_to_csv/requirements.txt

```
@@ -0,0 +1 @@
+pandas
```

We usually include `ipykernel` and `jupyter` here too so that if users spin up a fresh virtual environment it will be totally ready to go.
```suggestion
ipykernel
jupyter
pandas
```